web/src/main/webapp/WEB-INF/jsp/team/match/list.jsp

```diff
@@ -0,0 +1,99 @@
+<%@ page contentType="text/html;charset=UTF-8" pageEncoding="utf-8" trimDirectiveWhitespaces="true" session="false" %>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="my" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="s" uri="http://www.springframework.org/tags" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+
+<my:pagetemplate title="MatchesOverview">
+    <jsp:attribute name="body">
+         <div class="container">
+             <div class="row">
+                 <div class="col-md-2"></div>
```

Comment:
když chceš, aby ty "sloupce" byly odsazeny, tak se to nedělá tím že tam dáš prázdnej, ale na ten druhej dáš (v tomto případě) `offset-md-2`